crates/txtx-core/src/lib.rs

```diff
@@ -201,8 +201,8 @@ pub async fn start_supervised_runbook_runloop(
     loop {
         let event_opt = match action_item_responses_rx.try_recv() {
             Ok(action) => Some(action),
-            Err(TryRecvError::Empty) => None,
-            Err(TryRecvError::Disconnected) => return Ok(()),
```

Comment:
This also makes me nervous, mainly because I don't remember exactly why this stuff is the way it is, but I remember it being finicky to set up. Apologies, I should have documented things better